### PR TITLE
HW 2

### PR DESCRIPTION
**Pull request by kaalosaimi**
I faced difficulties in moving the box on the image to the right. It was very helpful for me to apply first a DOM, as it supported me in understanding how to structure the HTML and utilize CSS effectively

### CODE REVIEW

**Comment by kaalosaimi** [comment]
I faced difficulties in moving the box on the image to the right. It was very helpful for me to apply first a DOM, as it supported me in understanding how to structure the HTML and utilize CSS effectively

**Comment by kaalosaimi** [comment]
Quick update on the side box over the image

**Comment by kaalosaimi** [comment]
I was incorrect in choosing "float" property, it is updated to "right".

File: exercise/homework/main.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  border: none;
+  color: black;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  font-weight: bold;
+}
+.outterBox{
```
Comment (Samidahlawi) [inline exercise/homework/main.css:105]: I recommended you to add 'text-align: center;' to make the contents on the center.

**Comment by Samidahlawi** [comment]
Fantastic @khaled764 🤩. You're right is difficult but remember do not give up, the beginning is always the hardest, it will be more easy with time. By the way, your page looks nice. Good work @khaled764

**Pull request closed without merging**